config/src/config/execution_config.rs

```diff
@@ -54,6 +55,8 @@ impl Default for ExecutionConfig {
             service: ExecutionCorrectnessService::Thread,
             backend: SecureBackend::InMemoryStorage,
             sign_vote_proposal: true,
+            // Default value of 30 seconds for the network timeout.
+            network_timeout: 30,
```

Comment:
Yup, I realise this timeout covers multiple aspects and that's fine for now. I'd just like to avoid the case where we later add a `timeout_connect` option in ms, and then we have some timeouts in s and some in ms.

Reply:
@mgorven I'll make sure to get your thoughts on adding timeout to vault.  Better to get it close the first time from the handles that you'll need.

Reply:
This value seems problematic, shouldn't it be 30_000?

Reply:
Caught it.  Fixed all the other instances of 30 except the one that Michael actually commented on.